setup.py

```diff
@@ -582,7 +505,13 @@ def is_pure(self):
         return False
 
 
-helics_cli_install_requires = ["flask>=2", "requests", "flask-restful", "flask-cors", "pandas", "SQLAlchemy", "matplotlib"]
+helics_cli_install_requires = [
+    "requests",
+    "helics_cli_extras==0.0.1",
+    "pandas",
+    "SQLAlchemy",
+    "matplotlib",
+]
```

Comment:
Kind of surprising how many additional HELICS CLI dependencies there still are.

Reply:
The observer has some plotting and database functionality, which is where this comes from.

Reply:
Since we get rid of setup.py in the scikit-build-core PR, I don't think we need to update anything here. Since SQLAlchemy is already a dependency of the helics-cli-extras package, and all of the use of SQLAlchemy is confined to there, we should be able to remove it from the list of requirements on the main helics[cli] package (we might also consider updating the checks in https://github.com/GMLC-TDC/pyhelics/blob/jm/split_web/helics/cli.py#L59-L73 for flask and sqlalchemy to instead check the helics-cli-extras imports).

Reply:
Ok, fixed that too.